packages/babel-helper-simple-access/src/index.ts

```diff
@@ -20,73 +16,6 @@ export default function simplifyAccess(path: NodePath, bindingNames) {
 }
 
 const simpleAssignmentVisitor = {
-  UpdateExpression: {
```

Comment:
Yes.
What can I do? Should I bump the version of the package?

Reply:
No, because we keep all the versions synchronized (and it would require a major bump).

It's annoying, but a solution until Babel 8 is to pass a `includeUpdateExpression: boolean = true` option to `simplifyAccess`, and only skip the `UpdateExpression` visitor when that option is set to `false`. (and please add a comment to revisit it in Babel 8!)